src/coreclr/pal/src/thread/process.cpp

```diff
@@ -3286,13 +3286,19 @@ PROCAbortInitialize()
         char* dumpType = getenv("COMPlus_DbgMiniDumpType");
         char* diagStr = getenv("COMPlus_CreateDumpDiagnostics");
         BOOL diag = diagStr != nullptr && strcmp(diagStr, "1") == 0;
+        char* verboseStr = getenv("COMPlus_CreateDumpVerboseDiagnostics");
```

Comment:
Is there any particular reason that we haven't aligned these environment variables with the rest of our config variables (supporting a `DOTNET_` prefix in addition to a `COMPlus_` prefix)?

Reply:
It's been fixed, didn't make it into 6.0 https://github.com/dotnet/runtime/blob/a71d6f1c99595a6ceafe42ed8bc30fdf031e9b7e/src/coreclr/pal/src/thread/process.cpp#L3309 Any config in the PAL is not going to accept DOTNET_ in 6.0. Granted, it's very few

Reply:
Sorry I didn't realize this was going into 6.0. Sorry for the confusion.